test-integration/helpers/ui/test-element.coffee

```diff
@@ -94,10 +97,12 @@ class TestHelper extends TestItemHelper
       defaultWaitTime: 40 * 1000 # TODO: Letting tests define their own wait time is dangerous. tutor-dev takes > 10sec to delete a task-plan
       # defaultWaitTime: 20 * 1000 # 20sec seems to be enough for deployed code but not local
 
+    @test = test
     @_options = _.assign {}, defaultOptions, options
     @_el = {}
 
     commonElements.loadingState = @options.loadingLocator
+    commonElements.self = testElementLocator
```

Comment:
I like the `el.self()` syntax. Could it just happen to be one of the locators passed in (simplifying the constructor args) rather than special-casing it in here?